### DISCUSSION TITLE
Picviewer CE+下载图片后面总是出现;charset=utf-8

### DISCUSSION THREAD

**Discussion by nuomi092** [hoothin/UserScripts#426] [Show and tell]
Picviewer CE+在lofter下载图片后，图片格式后面总是出现;charset=utf-8，需要手动删除才行，在微博下载图片没有这个问题，不知道是什么原因，有解决办法吗？

**Comment by hoothin** [comment]
感谢反馈，最新版已经修复

**Reply by nuomi092** [reply]
好像是跟脚本或者拓展冲突了，我用的chorme还是这个问题，换了edge试了下可以正常下载出图片，感谢您的修复
